src/main/sensors/gyro.c

```diff
@@ -87,6 +87,7 @@ static FAST_RAM float accumulatedMeasurements[XYZ_AXIS_COUNT];
 static FAST_RAM float gyroPrevious[XYZ_AXIS_COUNT];
 static FAST_RAM timeUs_t accumulatedMeasurementTimeUs;
 static FAST_RAM timeUs_t accumulationLastTimeSampledUs;
+static float gyro_offset = 0;
```

Comment:
(`gyro_yaw_offset`)

Reply:
Gyro offset is different for each axis (caused mainly by manufacturing tolerance). Only yaw axis is important, roll and pitch are compensated from accelerometer.

Offset should be added only to yaw (adding it to roll/pitch gets compensated, so it did not have any visible influence)

Reply:
@ledvinap
changed to gyro_offset_yaw  (applies only to yaw axis)

Reply:
Sorry to pipe in late on this. But should an offset for roll and pitch be used in cases where the accelerometer is not enabled, or is this a case where enabling the accelerometer is the only viable course of action?

Reply:
@mikeller 
I am not sure ! 
I am reading Inversense mpu600  manual right now.
It seams that the offset is ADC related, it does not come from the MEMS (fabrication tolerance)
that's why I think it should be the same for all ADC channels !
@ledvinap thinks they are different !
The old offset correction was applied on all axes, now it's bound on yaw!

Reply:
@adrianmiriuta: I think that @ledvinap is correct that in the case when ACC is on, factoring in the offset for roll and pitch is not going to make a difference, since ACC will compensate for drift. And possibly 'enable ACC to get gyro roll / pitch compensation' is the right advice for this. I am just wondering if there are cases where this is not possible / desirable?

Reply:
@mikeller 
Yes that is correct  for ACC sensor  on.
What I ment is that 
I am not sure if we need different offsets for each axis in case ACC sensor is off.

Reply:
@mikeller 
@ledvinap 
ledvinap is correct each gyro axis has its own ADC.  That means the offsets are different!

I do not know if it is desirable to have  pitch and roll offset manual compensation.
I think now with conversion of gyro values to float the automatic adjustment should be sufficient.

Reply:
@mikeller : The drift is small and without ACC only ACRO mode is possible - pilot is far more than capable to compensate for drift.

Reply:
(@adrianmiriuta : Main part of offset is probably caused by gyro+pickup mechanical intolerances, not by ADC ...)

Reply:
@ledvinap: Good point about ACRO mode - drift there is too small to be picked up.